.circleci/config.yml

```diff
@@ -58,18 +58,18 @@ jobs:
     steps:
       - setup
       - run_with_pip_cache:
-          cache_key_base: lint-python3.8
+          cache_key_base: lint-python3.9
           steps:
-            - run: tox -f lint-ci
+            - run: tox -s false -f lint-ci
```

Comment:
Nit: When writing scripts or defining CI steps, consider using long-form command line flags for readability (I had to look up that this meant `--skip-missing-interpreters`).

Reply:
This is non-blocking feedback. There are lots of single-character flags used already, so you don't have to fix them all for this PR. It's okay to fix just this one, or plan to fix them all later in one fell swoop.

Reply:
👍 good point